src/app/containers/StoryPromo/index.jsx

```diff
@@ -1,25 +1,49 @@
 import React, { Fragment, useContext } from 'react';
-import { shape } from 'prop-types';
+import { shape, bool } from 'prop-types';
 import StoryPromoComponent, {
   Headline,
   Summary,
   Link,
 } from '@bbc/psammead-story-promo';
 import Timestamp from '@bbc/psammead-timestamp-container';
 import { storyItem } from '../../models/propTypes/storyItem';
-import StoryPromoFigure from './Figure';
+import FigureContainer from '../Figure';
+import Timestamp from '../Timestamp';
```

Comment:
I think this might have got confused in the latest pull, this Timestamp is declared above